### DISCUSSION TITLE
Pénzforgalmi elszámolású számlák WARNING

### DISCUSSION THREAD

**Discussion by tothj74** [nav-gov-hu/eVAT#14] [Kérdések és válaszok - Gépi interfész] [answered]
Kérdésem a pénzforgalmi elszámolású befogadott számlákra vonatkozik. 
Ha ezeket több részletben egyenlíti ki a vevő, akkor teljesen jogosan több  áfa időszakban is megjelenik majd a részösszegre vonatkozóan levonási joggal. Tervezik-e, hogy kezelik ezt az esetet, hogy ne kapjon WARNING-ot amiatt, hogy egy korábbi áfa időszakban már szerepel a számla?

**Comment by NTCA-tax** [comment, marked as answer]
Igen, ezt külön kell kezelnünk a warningoknál. A tervezet példálózó jelleggel felvet néhány warningot, de ez még nem a kész specifikáció. A tervezethez képest sokkal több warningot fogunk alkalmazni, amelyet átlátható módon fogunk dokumentálni.
